Add a note to set default values for custom resources (#4650)
godotengine/godot-docs: tutorials/scripting/resources.rst

    export(Resource) var sub_resource
    export(Array, String) var strings

>     # Make sure that every variable gets a default value. Otherwise, there will be problems with creating and editing your resource via the inspector.

Reviewer: ```suggestion
    # Make sure that every variable gets a default value.
    # Otherwise, there will be problems with creating and editing
    # your resource via the inspector.
```

    export(Resource) var sub_resource
    export(Array, String) var strings

>     # Make sure that every variable gets a default value. 
>     # Otherwise, there will be problems with creating and editing your resource via the inspector.

Reviewer: ```suggestion
    # Make sure that every parameter gets a default value. 
    # Otherwise, there will be problems with creating and editing
    # your resource via the inspector.
```

                [Export]
                public String[] Strings { get; set; }

>                 // Make sure that every variable gets a default value. Otherwise, there will be problems with creating and editing your resource via the inspector.

Reviewer: ```suggestion
                // Make sure that every variable gets a default value.
                // Otherwise, there will be problems with creating and editing
                // your resource via the inspector.
```

                [Export]
                public String[] Strings { get; set; }

>                 // Make sure that every variable gets a default value. 
>                 // Otherwise, there will be problems with creating and editing your resource via the inspector.

Reviewer: ```suggestion
                // Make sure that every parameter has a default value. 
                // Otherwise, there will be problems with creating and editing
                // your resource via the inspector.
```
